runtime/bindings/python/src/pyopenvino/core/infer_request.hpp

```diff
@@ -33,9 +36,10 @@ class InferRequestWrapper {
         return static_cast<double>(execTime.count()) * 0.000001;
     }
 
-    InferenceEngine::InferRequest _request;
-    InferenceEngine::ConstInputsDataMap _inputsInfo;
-    InferenceEngine::ConstOutputsDataMap _outputsInfo;
+    ov::runtime::InferRequest _request;
+    std::vector<ov::Output<const ov::Node>> _inputs;
+    std::vector<ov::Output<const ov::Node>> _outputs;
+
     Time::time_point _startTime;
     Time::time_point _endTime;
```

Comment:
Style fix: `_start_time` and `_end_time`

Reply:
Updated.